pyseir/inference/infer_rt.py

```diff
@@ -13,12 +13,6 @@
 from pyseir.utils import AggregationLevel, TimeseriesType
 from pyseir.utils import get_run_artifact_path, RunArtifact
 from pyseir.parameters.parameter_ensemble_generator import ParameterEnsembleGenerator
-from structlog.threadlocal import bind_threadlocal, clear_threadlocal, merge_threadlocal
-from structlog import configure
-from enum import Enum
-
-configure(processors=[merge_threadlocal, structlog.processors.KeyValueRenderer()])
-log = structlog.get_logger(__name__)
```

Comment:
Just flagging that I have no context on these changes.  So if you think they warrant further review, you should ping somebody else or we can chat through them.  If they're low-risk / already-discussed-with-folks / whatever, then cool.

Reply:
Yeah. These were already discussed promises I made to Tom last week and that he signed off on. I added in that code last week as a patch and how now cleaned it up.